libcodechecker/server/client_auth_handler.py

```diff
@@ -42,13 +43,23 @@ def getAuthParameters(self):
                                         True))
 
     @timeit
+    def getLoggedInUser(self):
+        if self.__auth_session:
+            return self.__auth_session.user
+        else:
+            return ""
+
+    @timeit
     def getAcceptedAuthMethods(self):
         return ["Username:Password"]
 
     @timeit
     def performLogin(self, auth_method, auth_string):
         if auth_method == "Username:Password":
-            authToken = self.__manager.create_or_get_session(auth_string)
+            session = self.__manager.create_or_get_session(auth_string)
+            authToken = None
```

Comment:
This variable is unused, and the lack of newline between the conditionals make the code hard to read. Could be simplified as:

```py
session = ...

if session:
    return session.token
else:
    raise ...
```